_docs/setup/target_rate.md

```diff
@@ -157,30 +157,30 @@ If we set no from/to times, then our 24 hour period being looked at ranges from
+| `2023-01-01T06:30` | `2023-01-01T06:30` - `2023-01-01T07:30` | `true`                             | The rate of 7 is in the past, so we must look for the next lowest combined rate. |![image](https://github.com/eugeneniemand/HomeAssistant-OctopusEnergy/assets/6813309/69f0525a-c9f1-4590-9243-309feff2d7fa)|
+| `2023-01-01T18:00` | `2023-01-01T18:00` - `2023-01-01T19:00` | `true`                             | The rate of 20 is in the past, so we must look for the next lowest combined rate which is 34. |![image](https://github.com/eugeneniemand/HomeAssistant-OctopusEnergy/assets/6813309/1dda9a65-5677-4c24-8a5d-5a8f496f84d6)|
 | `2023-01-01T18:30` | None | `true`                            | There is no longer enough time available within our restricted time, so we have to wait until our target period has passed to look at the next evaluation period. |
 
 If we set our from/to times to look over two days, from `20:00` to `06:00`, we then limit the period that we look at to overnight. The following table shows what this would be like.
 
-| current date/time  | period                                | `Re-evaluate multiple times a day` | reasoning |
-| ------------------ | ------------------------------------- | ---------------------------------- | --------- |
-| `2023-01-01T20:00` | `2023-01-01T23:30` - `2023-01-02T01:30` | `false`                            | Our lowest rate of 5 now falls between our overnight time period so is available. |
-| `2023-01-02T02:00` | `2023-01-01T23:30` - `2023-01-02T01:30` | `false`                            | Our lowest period is in the past, so we have to wait until our target period has passed to look at the next evaluation period. |
-| `2023-01-02T02:00` | `2023-01-02T04:30` - `2023-01-02T05:30` | `true`                             | The rate of 5 is in the past, so we must look for the next lowest combined rate, which includes our half hour rate at 7. |
+| current date/time  | period                                | `Re-evaluate multiple times a day` | reasoning | example |
+| ------------------ | ------------------------------------- | ---------------------------------- | --------- |---------|
+| `2023-01-01T20:00` | `2023-01-01T23:30` - `2023-01-02T01:30` | `false`                            | Our lowest rate of 5 now falls between our overnight time period so is available. |![image](https://github.com/eugeneniemand/HomeAssistant-OctopusEnergy/assets/6813309/9e213df8-4f37-4d5f-805d-0cfa4a5408c0)|
+| `2023-01-02T02:00` | `2023-01-01T23:30` - `2023-01-02T01:30` | `false`                            | Our lowest period is in the past, so we have to wait until our target period has passed to look at the next evaluation period. |![image](https://github.com/eugeneniemand/HomeAssistant-OctopusEnergy/assets/6813309/f1a20b31-3844-42fe-9805-42f5fda9a427)|
```

Comment:
I'm struggling to understand this one, I get the reasoning but I think the period is incorrect besides the fact that its also 2 hours instead of one but it seems to indicate the the period that is selected is in the past, should this not be a day ahead or None like the others?